packages/calcite-components/src/components/button/button.scss

```diff
@@ -506,12 +528,14 @@
 
     &:hover {
       --calcite-internal-button-shadow-color: var(--calcite-color-foreground-3);
+      stroke: var(--calcite-color-border-input);
```

Comment:
Should this be `border-color`? `stroke` is an [SVG prop](https://developer.mozilla.org/en-US/docs/Web/CSS/stroke). If so, can you set the internal border color CSS prop for consistency?